plugin/src/main/kotlin/com/github/triplet/gradle/play/tasks/internal/PlayPublishTaskBase.kt

```diff
@@ -27,6 +27,8 @@ abstract class PlayPublishTaskBase : DefaultTask(), ExtensionOptions {
 
     @get:Internal
     protected val publisher: AndroidPublisher by lazy {
+        val transport = transport
```

Comment:
Yeah, I'm scared that the truststore will be locked into the daemon. What if the dev changes their `gradle.properties` but we don't update the truststore? We could try resetting the transport at the end of the build, but that seemed messy.

WDYT?

Reply:
Yeah, we probably should try caching it, right?

Reply:
I believe we don't have to worry about that. This property is living inside each task. So it will be recreated for every invocation. This is different from the plugin class itself or the extension object

Reply:
That's incorrect: `transport` is in `Constants` which is alive for as long as the JVM lives.

Reply:
Good point. And I'll move it out of `Constants` since it's technically not a constant anymore. 👍